src/shogun2-core/src/main/java/de/terrestris/shogun2/service/PluginService.java

```diff
@@ -95,4 +146,14 @@ public String getPluginSource(String simpleClassName) throws Exception {
 	public void setDao(D dao) {
 		this.dao = dao;
 	}
+
+	/**
+	 * Sets the applicationService.
+	 *
+	 * @param applicationService
+	 */
+	@Autowired
```

Comment:
`setDao(D dao)` (a couple of lines above) has the `@autowired` as well. Is that beacuse it doesn't have the property `dao` itself, but inherits it?

I will change as suggested, though.